samples/AnomalyDetector/src/Generated/Models/ErrorResponse.cs

```diff
@@ -49,7 +49,7 @@ public partial class ErrorResponse
         /// <param name="code"> The error code. </param>
         /// <param name="message"> The message explaining the error reported by the service. </param>
         /// <exception cref="ArgumentNullException"> <paramref name="code"/> or <paramref name="message"/> is null. </exception>
-        public ErrorResponse(string code, string message)
+        internal ErrorResponse(string code, string message)
```

Comment:
there are a lot of similar changes like this in the regen. We need to know the exact reason of this change.

Reply:
Looks like the reason should be usage changed from RoundTrip to Output. but we need to know why because the description is not mentioning a change of usage